### ISSUE TITLE
Expo clipboard setString causes bouncing in ios Safari

### ISSUE THREAD

**Issue by christopherleja** [expo/expo#17337]
### Summary

When using expo clipboard's setString method in safari on ios, the screen bounces up and then down as the text area is added and removed. 

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Web

### SDK Version (managed workflow only)

44.0.6

### Environment

expo-env-info 1.0.3 environment info:
    System:
      OS: macOS 12.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
      Yarn: 1.22.18 - ~/.nvm/versions/node/v16.14.2/bin/yarn
      npm: 8.5.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.2, iOS 15.2, macOS 12.1, tvOS 15.2, watchOS 8.3
    IDEs:
      Xcode: 13.2.1/13C100 - /usr/bin/xcodebuild
    npmPackages:
      expo: ^44.0.0 => 44.0.4 
      react: 17.0.1 => 17.0.1 
      react-dom: 17.0.1 => 17.0.1 
      react-native: 0.64.3 => 0.64.3 
      react-native-web: 0.17.1 => 0.17.1 
    npmGlobalPackages:
      expo-cli: 5.3.0
    Expo Workflow: managed

### Reproducible demo

important note: the issue only appears when you access the snack in safari on ios--ie, go to the site and run the web preview in the safari browser on an iphone instead of in expo go. Press the Copy button and you should see the screen jump.

https://snack.expo.dev/@christopherleja/bossy-chocolates

**Comment by barthap** [comment]
Interesting issue. I think I have a workaround - to use the `AsyncClipboard` Web API:
```js
if (Platform.OS === 'web' && navigator.clipboard != null) {
  try { 
    await navigator.clipboard.writeText("hello world");
  } catch { 
    // this might fail when permission not granted (browser tab is not focused)
  }
} else {
  // this will be deprecated in SDK 45, use `await Clipboard.setStringAsync()` instead
  Clipboard.setString("hello world");
}
```

You can read more about this API [in this article](https://web.dev/async-clipboard/#writetext()). Expo Clipboard was recently migrated to this new API (will be released with Expo SDK 45, `expo-clipboard@^3.0.1`), except the `setString()`, because it could introduce some breaking changes, especially on Safari due to some limitations with asynchronous code.

> Note that this API require the `clipboard-write` browser permission. It is granted by default when the browser tab is currently active (focused).

Could you test if this works in your use case? I will then be more sure about implementing it in `expo-clipboard`.

**Comment by christopherleja** [comment]
Yes, I just tested it and it fixed the issue! Thank you so much!
